3-JSKata/2-strings-and-numbers/utilities.js

```
@@ -1,18 +1,66 @@
 
-function addStringsOrNumbers(a, b) {}
+  if (isNumber(a) && isNumber(b)) {
+    return num3
+  } else {
+    return String(num3)
+  }
+}
 
-function isEmail(str) {}
+function isEmail(str) {
+  for (let i = 0; i < str.length; i++) {
```

Biome (2.1.2)

[error] This code will never be reached ...

... because either this statement ...

... or this statement will return from the function beforehand


(lint/correctness/noUnreachable)